src/features/harborView/utils.ts

```diff
@@ -125,3 +127,16 @@ export const getMaps = (data: INDIVIDUAL_HARBOR | undefined): Map[] => {
     return acc;
   }, []);
 };
+
+export const priceTierToString = (priceTier: PriceTier | null): string => {
+  switch (priceTier) {
+    case PriceTier.TIER_1:
+      return '1';
+    case PriceTier.TIER_2:
+      return '2';
+    case PriceTier.TIER_3:
+      return '3';
+    default:
```

Comment:
Optional: could remove the default here so if anything changes it will be caught as an error during build.

Reply:
Changed. `PierProperties` is now rendered with `pier.priceTier !== null ? priceTierToString(pier.priceTier) : '-'` because `priceTier` is nullable in the API